src/de/mod.rs

```diff
@@ -506,10 +506,17 @@ impl<'a, 'de, 's> de::Deserializer<'de> for &'a mut Deserializer<'de, 's> {
                                     Some(b'r') => b'\r',
                                     Some(b't') => b'\t',
                                     Some(b'u') => {
+                                        // TODO - Replace with `<[u8]>::split_first_chunk::<4>` once MSRV >= 1.77
```

Comment:
Just bump the MSRV on the crate and use the new API. There's no reason we're holding 1.62 MSRV